packages/react-components/react-timepicker-compat/CHANGELOG.md

```diff
@@ -0,0 +1,107 @@
+# Change Log - @fluentui/react-timepicker-compat-preview
```

Comment:
we need to change this to `# Change Log - @fluentui/react-timepicker-compat`, otherwise users will read changelog of `react-timepicker-compat` for a different package which is confusing.


what should be done:

**Short version**

if there is only a consumers of this package that you're in direct contact

- copy only actual changes from this changelog avoiding those headers with compat-preview mentions
- change changefile description to patch -> merge -> release (0.0.0 -> 0.0.1 )
- create PR with current changefile description -> merge -> release   (0.0.1 -> 0.1.0)
- create PR that deletes `react-timepicker-compat-preview` from repository

**Long version**

if this package is used by various users we need to establish proper OSS release flow to avoid confusion:

- this changelog should be empty
- create PR that generates new blank `react-timepicker-compat`, merge -> release (0.0.0 -> 0.0.1)
- create PR that copies implementation from compat-preview to compat, merge -> release  (0.0.1 -> 0.0.2 )
- create PR that on will bump compat from `0.0.2 -> 0.1.0` 
- create PR that bump compat-preview  patch version with change -> this package is deprecated use react-timepicker-compat -> merge and release
- create PR that removes `compat-preview` from repository

Reply:
Thanks, I will start with another PR to create a blank compat package then.